charts/agent/charts/deployment-cluster-events/README.md

```diff
@@ -0,0 +1,96 @@
+# deployment-cluster-events
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.9.0](https://img.shields.io/badge/AppVersion-0.9.0-informational?style=flat-square)
+
+A Helm chart for Kubernetes
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://open-telemetry.github.io/opentelemetry-helm-charts | deployment-cluster-events(opentelemetry-collector) | 0.97.0 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| deployment-cluster-events.clusterRole.annotations | object | `{}` |  |
+| deployment-cluster-events.clusterRole.clusterRoleBinding.annotations | object | `{}` |  |
+| deployment-cluster-events.clusterRole.clusterRoleBinding.name | string | `""` |  |
+| deployment-cluster-events.clusterRole.create | bool | `true` |  |
+| deployment-cluster-events.clusterRole.name | string | `""` |  |
+| deployment-cluster-events.clusterRole.rules[0].apiGroups[0] | string | `""` |  |
```

Comment:
This seems a little painful...
I guess it's so that we allow customer to restrict the access for this agent easier?
(Either way this comment doesn't block this PR

Reply:
this is just the schema for the `otel-collector` open source chart